tests/testExamples/Simple_test_before_feature.js

```diff
@@ -15,6 +15,7 @@ module.exports = function (dslName) {
                     }
                 ]
             },
+            is_favorite: false,
```

Comment:
if is_favorite is not mandatory, i suggest reverting 
tests/testExamples/Simple_test_before_feature.js
tests/testExamples/Simple_test.js
to verify backward compatible